R/run.R

```diff
@@ -318,6 +320,7 @@ CmdStanRun$set("private", name = "run_sample_", value = .run_sample)
 }
 CmdStanRun$set("private", name = "run_optimize_", value = .run_other)
 CmdStanRun$set("private", name = "run_variational_", value = .run_other)
+CmdStanRun$set("private", name = "run_generate_quantities_", value = .run_other)
```

Comment:
I can't comment on the exact code but there is also the question of how to run generate quantities if you have multiple CSV files for a fit object with multiple chains.
Options:
- run multiple processes (1 chain - 1 process). So basically similar to what we do for sample.
- run one process 1 chain at a time

Reply:
Generate quantities are quick in general so we maybe start with the sequential version and then upgrade later?

Reply:
My initial though is that we should use multiple processes (if the user wants). Is there a good reason to avoid that?

Reply:
Not really no. Can we use the name chains and parallel chains in this context? That would make things a lot easier.

Reply:
Yeah, let's just use the same names and clarify in the doc that it's technically not MCMC chains.